worlds/kh1/Items.py

```diff
@@ -0,0 +1,534 @@
+   #"Furious Bellow":                    KH1ItemData("Abilities",        code = 264_3048, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+   #"Spiral Wave":                       KH1ItemData("Abilities",        code = 264_3049, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+   #"Thunder Potion":                    KH1ItemData("Abilities",        code = 264_3050, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+   #"Cure Potion":                       KH1ItemData("Abilities",        code = 264_3051, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+   #"Aero Potion":                       KH1ItemData("Abilities",        code = 264_3052, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Slapshot":                          KH1ItemData("Abilities",        code = 264_3053, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Sliding Dash":                      KH1ItemData("Abilities",        code = 264_3054, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Hurricane Blast":                   KH1ItemData("Abilities",        code = 264_3055, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Ripple Drive":                      KH1ItemData("Abilities",        code = 264_3056, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Stun Impact":                       KH1ItemData("Abilities",        code = 264_3057, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Gravity Break":                     KH1ItemData("Abilities",        code = 264_3058, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Zantetsuken":                       KH1ItemData("Abilities",        code = 264_3059, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Tech Boost":                        KH1ItemData("Abilities",        code = 264_3060, classification = ItemClassification.useful,      max_quantity = 4,  weight = 10),
+    "Encounter Plus":                    KH1ItemData("Abilities",        code = 264_3061, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "Leaf Bracer":                       KH1ItemData("Abilities",        code = 264_3062, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+   #"Evolution":                         KH1ItemData("Abilities",        code = 264_3063, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
+    "EXP Zero":                          KH1ItemData("EXP Zero",         code = 264_3064, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
```

Comment:
```suggestion
    "EXP Zero":                          KH1ItemData("Abilities",         code = 264_3064, classification = ItemClassification.useful,      max_quantity = 1,  weight = 10),
```

Probably this was the intent?

Reply:
This was intentional to not shuffle in EXP Zero when generation pulls abilities